share/OpenMS/SCHEMAS/pepXML_v122.xsd

```diff
@@ -1,1770 +1,2007 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema targetNamespace="http://regis-web.systemsbiology.net/pepXML" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://regis-web.systemsbiology.net/pepXML" xmlns:pepx="http://regis-web.systemsbiology.net/pepXML" elementFormDefault="qualified">
```

Comment:
I believe this file needs to remain unchanged. But somehow spacings are changed that it looks like whole file is changed, even though it's not. I think it's better to revert this file to the previous commit.
i.e., git checkout f3fb2334890a96132e53a38741756c7691c0e0f1 -- share/OpenMS/SCHEMAS/pepXML_v122.xsd
then maybe try git push --force

Reply:
Thanks.